drivers/azure/azure.go

```diff
@@ -35,10 +36,13 @@ const (
 	flAzureSubscriptionID  = "azure-subscription-id"
 	flAzureResourceGroup   = "azure-resource-group"
 	flAzureSSHUser         = "azure-ssh-user"
+	flAzureWinRMUser       = "azure-winrm-user"
+	flAzureWinRMPassword   = "azure-winrm-password"
 	flAzureDockerPort      = "azure-docker-port"
 	flAzureLocation        = "azure-location"
 	flAzureSize            = "azure-size"
 	flAzureImage           = "azure-image"
+	flAzureOS              = "azure-os"
```

Comment:
I'd say let's group flags for windows together it'll make it easy to refactor to other drivers tomorrow